src/platforms/esp32/components/avm_builtins/gpio_driver.c

```diff
@@ -181,7 +214,17 @@ static inline term hold_en(term gpio_num_term)
 
 static inline term hold_dis(term gpio_num_term)
 {
-    int gpio_num = term_to_int(gpio_num_term);
+    gpio_num_t gpio_num;
+    if (LIKELY(term_is_integer(gpio_num_term))) {
+        avm_int_t pin_int =  term_to_int32(gpio_num_term);
```

Comment:
the extra space has been added by mistake several times.

Reply:
Should our check formatting workflow have caught these mistakes?